packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -444,11 +472,62 @@ export default class QuanticCategoryFacet extends LightningElement {
     return path.join('/');
   }
 
-  setFocusToFirstFacetValue() {
+  /**
+   * Sets the focus on the target element.
+   */
+  setFocusOnTarget() {
+    this.focusShouldBeInFacet = false;
+    if (!this.focusTarget) {
+      return;
+    }
+    // eslint-disable-next-line default-case
+    switch (this.focusTarget.type) {
+      case 'facetHeader':
+        this.setFocusOnHeader();
+        return;
+      case 'facetValue':
+        if (this.values.length) {
+          this.setFocusOnFirstFacetValue();
+        } else {
+          this.setFocusOnLastNonActiveParent();
+        }
+    }
```

Comment:
```suggestion
    if(this.focusTarget.type === 'facetHeader') {
        this.setFocusOnHeader();
    } else if (this.focusTarget.type === 'facetValue') {
         if (this.values.length) {
          this.setFocusOnFirstFacetValue();
        } else {
          this.setFocusOnLastNonActiveParent();
        }
    }
```

Simplifies it a bit and removes the need for the switch, and the eslint-disable